docs/elasticsearch/index-alias.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: default
+title: Index Alias
+parent: Elasticsearch
+nav_order: 5
+---
+
+# Index alias
+
+An alias is a virtual index name that can point to one or more indices.
+
+If your data is spread across multiple indices, rather than keeping track of which indices to query, you can create an alias and query that instead.
+
+For example, if you’re storing logs into indices based on the month and you frequently query the logs for the previous two months, you can create a `last_2_months` alias and update the indices it points to each month.
+
+Since you can switch the indices an alias points to at any time, referring to indices using aliases in your applications allows you to reindex your data without any downtime.
+
+Index aliases are point-in-time based so they only point to the current index. Aliases are NOT automatically updated when indices that are added to an alias are changed later.
```

Comment:
based, so they only

Aliases are *not* automatically updated

I don't understand what you're saying in the second sentence. Can you tweak the language to clarify?

Reply:
Removed this note because any changes to an index after it's been added to an alias does reflect in the alias.